<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slitttranslationhorizontalpanel</class>
 <widget class="QWidget" name="slitttranslationhorizontalpanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>480</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Pinhole</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCALed" name="LedV1_3" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_getn.E.</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save all as startup</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_AKTION</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>In</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_AKTION</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>DoRef</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_AKTION</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Hard (H)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_REF_STAT</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Out</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_AKTION</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>92</y>
     <width>269</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCALabel" name="label_3">
      <property name="text">
       <string>center</string>
      </property>
      <property name="pv" stdset="0">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>48</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCAPushButton" name="pushButton">
      <property name="text">
       <string>Goto</string>
      </property>
      <property name="pv" stdset="0">
       <string>u411pgm1:PH_1_GON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCAPushButton" name="pushButton_2">
      <property name="text">
       <string>Save as</string>
      </property>
      <property name="pv" stdset="0">
       <string>u411pgm1:PH_1_SAVEN</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>168</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCAPushButton" name="pushButton_3">
      <property name="text">
       <string>Goto</string>
      </property>
      <property name="pv" stdset="0">
       <string>u411pgm1:PH_1_GON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCAPushButton" name="pushButton_4">
      <property name="text">
       <string>Save as</string>
      </property>
      <property name="pv" stdset="0">
       <string>u411pgm1:PH_1_SAVEN</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>168</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QCAPushButton" name="pushButton_7">
      <property name="text">
       <string>Goto</string>
      </property>
      <property name="pv" stdset="0">
       <string>u411pgm1:PH_1_GON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCAPushButton" name="pushButton_8">
      <property name="text">
       <string>Save as</string>
      </property>
      <property name="pv" stdset="0">
       <string>u411pgm1:PH_1_SAVEN</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCALabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>500</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>center</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_GET_IN</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>500</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_AKTION</string>
   </property>
  </widget>
  <widget class="QCALed" name="LedV1_2" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_getn.D.</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>In</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_DOSTEP</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:MonoName</string>
   </property>
  </widget>
  <widget class="QCALed" name="LedV1" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_getn.C.</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>168</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCAPushButton" name="pushButton_5">
      <property name="text">
       <string>Goto</string>
      </property>
      <property name="pv" stdset="0">
       <string>u411pgm1:PH_1_GON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCAPushButton" name="pushButton_6">
      <property name="text">
       <string>Save as</string>
      </property>
      <property name="pv" stdset="0">
       <string>u411pgm1:PH_1_SAVEN</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCAPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Out</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_DOSTEP</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_Name</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="goto1axis" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>350</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Goto</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:PH_1_SET</string>
   </property>
   <property name="units" stdset="0">
    <string comment="mm/s">um</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_GET</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="step1saxis" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>350</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Rel.Steps</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:PH_1_SETSTEP</string>
   </property>
   <property name="units" stdset="0">
    <string>um</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_STEP</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="speed1axis" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>350</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>jogSpeed</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:PH_1_SETJOGSPEED</string>
   </property>
   <property name="units" stdset="0">
    <string>mm/s</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_JOGSPEED</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="po1saxis" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>350</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>setpos</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:PH_1_Load</string>
   </property>
   <property name="units" stdset="0">
    <string comment="mm/s">um</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_Load</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCALabel</class>
   <extends>QLabel</extends>
   <header>qcalabel.h</header>
  </customwidget>
  <customwidget>
   <class>QCALed</class>
   <extends>QWidget</extends>
   <header>qcaled.h</header>
  </customwidget>
  <customwidget>
   <class>QCAAxis</class>
   <extends>QWidget</extends>
   <header>qcaaxis.h</header>
  </customwidget>
  <customwidget>
   <class>QCAPushButton</class>
   <extends>QPushButton</extends>
   <header>qcapushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
